monkey/tests/unit_tests/agent_plugins/exploiters/mssql/test_mssql_plugin.py

```diff
@@ -122,7 +122,7 @@ def test_run__attempts_exploit_if_port_status_unknown(
     target_host: TargetHost,
     tcp_port_status: PortScanDataDict,
 ):
-    host = target_host
+    host = target_host.model_copy(deep=True)
```

Comment:
`target_host` is a function-scoped fixture, which should mean that each test has its own copy already. Furthermore, exploiters shouldn't be modifying the `target_host` object. Why is this change needed?